source/Table/Column.js

```diff
@@ -77,3 +67,14 @@ export default class Column extends Component {
     width: PropTypes.number.isRequired
   }
 }
+
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
This should be wrapping `propTypes` not `defaultProps`. Those should stay in the class.